src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-types/lancie-admin-ticket-options.html

```diff
@@ -0,0 +1,164 @@
+<link rel="import" href="../../../../bower_components/neon-animation/web-animations.html">
+<link rel="import" href="../../../../bower_components/paper-listbox/paper-listbox.html">
+<link rel="import" href="../../../../bower_components/paper-item/paper-item.html">
+
+<link rel="import" href="../../lancie-admin-page-layout.html">
+<link rel="import" href="../../../lancie-admin-table/lancie-admin-table.html">
+
+<link rel="import" href="lancie-admin-ticket-option.html">
+
+<dom-module id="lancie-admin-ticket-options">
+  <template>
+    <style include="iron-flex iron-flex-alignment"></style>
+    <style>
+      :host {
+        display: block;
+      }
+
+      paper-card {
+        width: calc(100% - 32px);
+        margin: 16px;
+      }
+
+      [hidden] {
+        display: none !important;
+      }
+
+      @media (max-width: 640px) {
+        paper-card {
+          width: 100%;
+          margin: 0;
+        }
+      }
+    </style>
+
+    <lancie-ajax auto-fire id="getOptionAjax" refurl="tickets/options" on-lancie-ajax="onOptionResponse"></lancie-ajax>
+
+    <lancie-ajax id="ajaxDeleteTicketOption" method="DELETE" refurl="tickets/options/[[deleteOption.id]]" on-lancie-ajax="onDeleteResponse"></lancie-ajax>
+
+    <paper-card heading="Ticket Options">
+      <div class="card-content" on-delete-ticket-option-dialog="deleteTicketOptionDialog">
+        <lancie-error id="optionError"></lancie-error>
+        <template is="dom-repeat" items="{{options}}">
+          <lancie-admin-ticket-option option="[[item]]"></lancie-admin-ticket-option>
+        </template>
+      </div>
+      <div class="card-actions">
+        <paper-button on-tap="openAddOptionDialog">Add Ticket Option</paper-button>
+      </div> 
+    </paper-card>
+    <lancie-dialog id="addTicketOptionDialog">
+      <h2>Add Ticket Option</h2>
+      <div class="dialog-content">
+        <lancie-error id="addDialogError"></lancie-error>
+        <lancie-form id="addTicketOptionForm" refurl="tickets/options" on-response="onAddOptionResponse">
+          <paper-input label="Name" name="name" auto-validate required error-message="This field is required!"></paper-input>
+          <paper-input label="Price" name="price" step=".01" type="number" auto-validate required error-message="This field is required!"></paper-input>
+        </lancie-form>
+      </div>
+      <div class="dialog-actions">
+        <paper-button on-tap="tryAddOption">Add</paper-button>
+      </div>
+    </lancie-dialog>
+
+    <lancie-dialog id="deleteTicketOptionDialog">
+      <h2>Confirm deletion</h2>
+      <div class="dialog-content">
+        <lancie-error id="dialogError"></lancie-error>
+        <h4>Are you certain you want to delete this ticket option?</h4>
+        <p>[[deleteOption.name]]</p>
+      </div>
+      <div class="dialog-actions">
+        <paper-button dialog-dismiss>Cancel</paper-button>
+        <paper-button on-tap="tryOptionDelete" class="confirm-delete-button">Delete</paper-button>
```

Comment:
`tryDeleteOption` maybe? That would be in line with the naming for `tryAddOption`.